app/views/layouts/application.html.erb

```diff
@@ -67,6 +65,12 @@
   </head>
 
   <body class="<%= body_classes %>">
+    <div data-assets>
+      <div data-assets-css>
+        <%= render partial: "common/discourse_stylesheet" %>
+      </div>
+    </div>
```

Comment:
I like the idea of wrapping these! But a `<div>` is semantically not the correct element type. How about something like:

```suggestion
    <discourse-assets>
      <discourse-assets-stylesheets>
        <%= render partial: "common/discourse_stylesheet" %>
      </discourse-assets-stylesheets>
    </discourse-assets>
```

Reply:
Yeah! I like that better :+1: